akka-http-core/src/main/scala/akka/http/scaladsl/model/FormData.scala

```diff
@@ -13,11 +13,11 @@ import akka.http.scaladsl.model.MediaTypes._
  */
 final case class FormData(fields: Uri.Query) {
   def toEntity: akka.http.scaladsl.model.RequestEntity =
-    toEntity(HttpCharsets.`UTF-8`)
+    toEntity(`application/x-www-form-urlencoded`.charset)
 
   def toEntity(charset: HttpCharset): akka.http.scaladsl.model.RequestEntity = {
```

Comment:
We might deprecate `toEntity(HttpCharset)` as `FormData` has fixed charset UTF-8 now. WDYT?

Reply:
Sounds good.